.github/workflows/stale-bot.yml

```diff
@@ -0,0 +1,15 @@
+name: "Close stale PRs"
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
```

Comment:
you might need these permissions for being able to add PR labels:

```
permissions:
  pull-requests: write
```

Reply:
Yes you're right!
Done.